.gitignore

```diff
@@ -9,7 +11,6 @@
 
 *.iml
 .gradle
-.idea
```

Comment:
why?

Reply:
The PR that is Submitted almost 10 days ago was included but at the latest, it was removed. It was submitted by mistake